rust/kcl-lib/src/std/mod.rs

```diff
@@ -285,18 +286,25 @@ const DEFAULT_TOLERANCE: f64 = 0.0000001;
 
 /// Compute the length of the given leg.
 pub async fn leg_length(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
-    let (hypotenuse, leg, ty) = args.get_hypotenuse_leg()?;
-    let result = inner_leg_length(hypotenuse, leg);
-    Ok(KclValue::from_number_with_type(result, ty, vec![args.into()]))
+    let hypotenuse: TyF64 = args.get_kw_arg("hypotenuse")?;
```

Comment:
Could you use the typed versions of `get...` here (and below) so numbers will get properly converted

Reply:
What type should it be looking for? Just any number?

Reply:
RuntimeType::length() for both, and then use NumericType::combine_eq_coerce to make sure they are in the same units